apicast/src/resty/http_ng.lua

```diff
@@ -87,7 +87,7 @@ http.method = function(method, client)
     local req_params = get_request_params(method, client, url, options)
     local req = http.request.new(req_params)
 
-    return client.backend.send(req)
+    return client.backend:send(req)
```

Comment:
Hah! `:` is syntactic sugar for passing self.

So these are the same code:

```lua
object:method(param)
object.method(object, param)
```

This works in the function definition too:

```lua

function _M.method(self, param)
end

function _M:method(param)
end
```

Passing `self` It is the only way how a function can know about the object being called on.